omod/src/main/webapp/resources/scripts/metadata/manageRoles.js

```diff
@@ -122,12 +122,28 @@ angular.module("manageRoles", [ "roleService", "privilegeService", "ngDialog", "
                         $scope.privileges = results;
                         $scope.inheritedPrivilegeFlags = Array($scope.privileges.length);
                         $scope.privilegeFlags = Array($scope.privileges.length);
-                                                
+                        filterPrivileges();                       
                         loadInheritedPrivileges();
 
                     });
                 });
             }
+            //Filters Privileges to required criteria. ie Only "Application privileges" are required.
+            function filterPrivileges() {
+            	var inheritedPrivileges =[];
```

Comment:
I think you are using the wrong name here, these aren't inherited privileges, they are application privileges